sims/verisim/verilator.mk

```diff
@@ -1,7 +1,17 @@
-# Build and install our own Verilator, to work around versionining issues.
-VERILATOR_VERSION=3.920
+#########################################################################################
+# verilator installation makefrag
+#########################################################################################
+
+#########################################################################################
+# verilator version, binary, and path
+#########################################################################################
+VERILATOR_VERSION=4.008
```

Comment:
Is this verilator versiion good for people? It matches RC's version. (Note: Multithreading support is not working with RC and Boom currently but this change is keeping single-threaded support)

Reply:
Yes matching RC is good.